wallet.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package rusk;
+option go_package = "github.com/dusk-network/rusk-schema;rusk";
+
+message EmptyRequest {}
```

Comment:
It's true that Rusk manages the Phoenix-based logic. However, I think we are storing the keys on `dusk-blockchain`, so for now, maybe expand this to be something like:

```
message GetBalanceRequest {
    ViewKey vk = 1;
}
```

Reply:
In former version `ViewKey` was stored in the wallet file, I'm not sure if we need it here but let's add it for now

Reply:
As far as I understand, we do not let Rusk read any of the disk stuff related to keys and personal notes, as that is not its responsibility.